lib/rust/ensogl/core/src/debug/monitor.rs

```diff
@@ -402,11 +401,6 @@ impl Panel {
         self.rc.borrow_mut().draw(dom)
     }
 
-    /// Start measuring the data.
-    pub fn begin(&self) {
-        self.rc.borrow_mut().begin()
-    }
-
```

Comment:
You removed the function `begin` which was described as "Start measuring the data.". You moved the measurements out of Panel. You left the function `end` with the description "Stop measuring the data.". This is bad:

1. Panel is not measuring data anymore, so this is confusing for the reader.
2. The function `end` is not named correctly, because it doesn't provide the reader with enough semantic information – what does "end" mean if there was never "start"?

Please always change the naming and documentation of functions in structures that you change, so they reflect the changes.

Reply:
I changed the function name to `sample_and_postprocess()`, using the meaning of "sample" as a verb (as in: "to sample"); however, I also considered `fetch_and_postprocess()` - would you maybe find that alternative (or some other one) easier to understand here?

Reply:
I like these names! Currently I cannot think of anything better.